### PR TITLE
[BACKLOG-15218] Created default configuration for viz api views

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pentaho/millenniumfalcon @diogofscmariano please review.

We already found some issues related to multi charts and scrolling.

### CODE REVIEW

File: impl/client/src/test/javascript/pentaho/visual/color/paletteRegistry.spec.js [hunk 1]
```diff
@@ -242,25 +242,29 @@ define(["pentaho/visual/color/paletteRegistry"], function(singletonPaletteRegist
...
 
-    it("should be pre-loaded with eleven palettes", function() {
+    it("should be pre-loaded with twelve palettes", function() {
 
       var palettes = singletonPaletteRegistry.getAll();
       expect(palettes instanceof Array).toBe(true);
 
-      // TODO: Temporarily, 15 palettes (and not 11) due to the palettes registered by CDF.
-      expect(palettes.length).toBe(15);
+      // TODO: Temporarily, 16 palettes (and not 16) due to the palettes registered by CDF.
```
Comment (nantunes) [inline impl/client/src/test/javascript/pentaho/visual/color/paletteRegistry.spec.js]: So, when will they stop being 16 and return to 16?

Comment (dcleao) [inline impl/client/src/test/javascript/pentaho/visual/color/paletteRegistry.spec.js]: When CDF work by Diogo stops including more palettes in a strange way.

File: impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js [hunk 2]
```diff
@@ -0,0 +1,601 @@
...
+  var defaultVizApiPalette = "viz_api_all_colors";
+  paletteRegistry.setDefault(defaultVizApiPalette);
+
+  return {
+    rules: [
+      // Pentaho CCC Abstract
+      {
+        select: {
+          priority: -1,
+          // application: "pentaho-det",
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js]: Should be removed, not commented, no?

File: impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js [hunk 3]
```diff
@@ -0,0 +1,601 @@
...
+
+      numberFormat = pvc.data.numberFormat(mask);
+      numberFormat.style(numberStyle);
+      numberFormatCache[key] = numberFormat;
+    }
+
+    return numberFormat;
+  }
+
+  function configureVizAPIPalette() {
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js]: This Viz palette stuff is already in the color palette registry and should be removed from here

Comment (dcleao) [inline impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js]: And remember to fix tests to # 15 palettes ?

File: impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js (hunk 2 above)
Comment (davidmsantos90) [inline impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js]: yes, they are now removed :+1:

File: impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js (hunk 3 above)
Comment (davidmsantos90) [inline impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js]: I forgot to delete this code, but as the comments, it is now gone :+1:
and the test is also fixed

**Comment by davidmsantos90** [comment]
@pentaho/millenniumfalcon updated the PR with you reviews input. please review

**Pull request merged**